Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml

```diff
@@ -72,6 +72,59 @@
     rewardMinAmount: 0
     rewardMaxAmount: 0
     possibleRewards:
+    - Basketball
+    - BalloonNT
+    - BalloonCorgi
+    - BoxDonkSoftBox
+    - BoxCartridgeCap
+    - BeachBall
+    - CandyBucket
+    - CrayonBox
+    - ClothingHeadHatCowboyRed
+    - FoamCrossbow
+    - FoamBlade
+    - FoamCutlass
+    - Football
+    - GlowstickBase #green
+    - GlowstickBlue
+    - GlowstickYellow
+    - GlowstickPurple
+    - GlowstickRed
+    - HarmonicaInstrument
+    - OcarinaInstrument
+    - RecorderInstrument
+    - GunpetInstrument
+    - BirdToyInstrument
+    - MysteryFigureBox
+    - PlushieHampter
+    - PlushieLizard
+    - PlushieAtmosian
+    - PlushieSpaceLizard
+    - PlushieNuke
+    - PlushieCarp
+    - PlushieMagicarp
+    - PlushieHolocarp
+    - PlushieRainbowCarp
+    - PlushieRatvar
+    - PlushieNar
+    - PlushieSnake
+    - PlushieArachind
+    - PlushieMoth
+    - PlushieHampter
+    - PlushiePenguin
+    - PlushieHuman
+    - PlushieRouny
+    - PlushieBee
+    - PlushieSlime
+    - PlushieGhost
+    - PlushieDiona
+    - PlushieSharkBlue
+    - PlushieVox
+    - PlushieXeno
+    - PlasticBanana
+    - RevolverCapGun
+    - SnapPopBox
+    - TeslaToy
```

Comment:
remove this one probably

Reply:
you want the tesla gone too?

Reply:
just the tesla or all 4 or them?

Reply:
I removed the tesla, but a cupgun, snap pops, and a plastic banana are all things ive received from real life arcades so I think they'd be appropriate to keep. also the capgun was already a reward on the list.

Reply:
> I removed the tesla, but a capgun, snap pops, and a plastic banana are all things ive received from real life arcades so I think they'd be appropriate to keep. also the capgun was already a reward on the list.

thats fine it was just the tesla

Reply:
The plastic banana is a plastic explosive iirc

Reply:
> The plastic banana is a plastic explosive iirc

you can get it from toy crate so idk

Reply:
I think this is the correct ID for the actual plastic toy banana. I think the explosive banana ID is something like explosivebanana but I dont rememeber